v2/users.go

```diff
@@ -0,0 +1,98 @@
+package tsclient
+
+import (
+	"context"
+	"net/http"
+	"time"
+)
+
+const (
```

Comment:
Oh nit - maybe we should add docs to all of these consts since they're exported

Reply:
I'm not entirely sure what to put here, as the [api doc](https://tailscale.com/api#model/user) doesn't provide any details either.